coderedcms/utils.py

```diff
@@ -37,3 +37,18 @@ def fix_ical_datetime_format(dt_str):
         dt_str = dt_str[:-3] + dt_str[-2:]
         return dt_str
     return dt_str
+
+def convert_to_amp(value):
+    """
+    Function that converts non-amp compliant html to valid amp html.
+    """
+    soup = BeautifulSoup(value)
+
+    #Replace img tags with amp-img
+    try:
+        img_tags = soup.find('img')
+        img_tags.name = 'amp-img'
```

Comment:
One item of note, amp image tags must have a width and height specified - not sure how the richtext filter outputs them.

Reply:
The richtext filter outputs the width and height.